App.tsx

```diff
@@ -29,6 +29,10 @@ export type RootStackParamList = {
 	Map: undefined;
 	InventoryModification: { bot: Bot };
 	AddItem: { bot: Bot };
+	ItemWeight: {
+		itemId: string;
+		botId: string;
```

Comment:
I think we want to keep it consistent right, so instead of botId, we should pass in Bot